### ISSUE TITLE
pacaur -Syu --devel fails to update packages

### ISSUE THREAD

**Issue by ChuckDaniels87** [rmarquis/pacaur#480]
I've tried to update my _devel packages_ but it fails to install them. 
**pacaur version:**  *4.6.0*

All the process is correct (fetching, building, compressing, ...) but pacaur tries to install always the same package. I've used the command `pacaur -Syu --devel`.

------
Log:
```
:: Starting AUR upgrade...
:: resolving dependencies...
:: looking for inter-conflicts...
:: hardcode-fixer-git latest revision -- fetching
:: numix-circle-icon-theme-git latest revision -- fetching
:: numix-icon-theme-git latest revision -- fetching
:: oh-my-zsh-git latest revision -- fetching
:: powerline-fonts-git latest revision -- fetching
:: vundle-git latest revision -- fetching

AUR Packages  (6):

Name                             Old Version           New Version         

aur/hardcode-fixer-git           1:r442.d6c80a7-1      latest                             
aur/numix-circle-icon-theme-git  0.r3033.e179931-1     latest                             
aur/numix-icon-theme-git         0.r1763.292f2b2-1     latest                             
aur/oh-my-zsh-git                3859.bd6dbd1-1        latest                             
aur/powerline-fonts-git          r100.a44abd0-2        latest                             
aur/vundle-git                   0.10.2.603.4984767-1  latest                             

:: Proceed with installation? [Y/n] y
:: Retrieving package(s)...
:: hardcode-fixer-git build files are up-to-date -- skipping
:: numix-circle-icon-theme-git build files are up-to-date -- skipping
:: oh-my-zsh-git build files are up-to-date -- skipping
:: powerline-fonts-git build files are up-to-date -- skipping
:: vundle-git build files are up-to-date -- skipping
:: numix-icon-theme-git build files are up-to-date -- skipping
:: Checking numix-icon-theme-git integrity...
==> Making package: numix-icon-theme-git 0.r317.290d12e-1 (dom abr 10 01:31:45 CEST 2016)
==> Retrieving sources...
  -> Updating numix-icon-theme git repo...
Fetching origin
==> Validating source files with sha256sums...
    numix-icon-theme ... Skipped
:: Checking vundle-git integrity...
==> Making package: vundle-git 0.10.2.580.cfd3b2d-1 (dom abr 10 01:31:50 CEST 2016)
==> Retrieving sources...
  -> Updating vundle-git git repo...
Fetching origin
  -> Found vundle.vimrc
==> Validating source files with sha512sums...
    vundle-git ... Skipped
    vundle.vimrc ... Passed
:: Checking powerline-fonts-git integrity...
==> Making package: powerline-fonts-git r100.a44abd0-2 (dom abr 10 01:31:51 CEST 2016)
==> Retrieving sources...
  -> Updating fonts git repo...
Fetching origin
==> Validating source files with md5sums...
    fonts ... Skipped
:: Checking oh-my-zsh-git integrity...
==> Making package: oh-my-zsh-git 3143.4224c2a-1 (dom abr 10 01:31:53 CEST 2016)
==> Retrieving sources...
  -> Updating oh-my-zsh-git git repo...
Fetching origin
==> Validating source files with md5sums...
    oh-my-zsh-git ... Skipped
:: Checking numix-circle-icon-theme-git integrity...
==> Making package: numix-circle-icon-theme-git 0.r1677.fe74d83-1 (dom abr 10 01:31:56 CEST 2016)
==> Retrieving sources...
  -> Updating numix-circle-icon-theme git repo...
Fetching origin
==> Validating source files with sha256sums...
    numix-circle-icon-theme ... Skipped
:: Checking hardcode-fixer-git integrity...
==> Making package: hardcode-fixer-git 1:r395.a01f915-1 (dom abr 10 01:31:58 CEST 2016)
==> Retrieving sources...
  -> Updating hardcode-fixer-git git repo...
Fetching origin
==> Validating source files with md5sums...
    hardcode-fixer-git ... Skipped
:: Building numix-icon-theme-git package(s)...
==> Making package: numix-icon-theme-git 0.r1763.292f2b2-1 (dom abr 10 01:31:59 CEST 2016)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating numix-icon-theme git repo...
Fetching origin
==> Validating source files with sha256sums...
    numix-icon-theme ... Skipped
==> Extracting sources...
  -> Creating working copy of numix-icon-theme git repo...
Switched to a new branch 'makepkg'
==> Starting pkgver()...
==> Entering fakeroot environment...
==> Starting package()...
==> Tidying install...
  -> Removing libtool files...
  -> Purging unwanted files...
  -> Removing static library files...
  -> Compressing man and info pages...
==> Checking for packaging issue...
==> Creating package "numix-icon-theme-git"...
  -> Generating .PKGINFO file...
  -> Generating .BUILDINFO file...
  -> Adding install file...
  -> Generating .MTREE file...
  -> Compressing package...
==> Leaving fakeroot environment.
==> Finished making: numix-icon-theme-git 0.r1763.292f2b2-1 (dom abr 10 01:32:08 CEST 2016)
==> Cleaning up...
:: Installing numix-icon-theme-git package(s)...
loading packages...
warning: numix-icon-theme-git-0.r1763.292f2b2-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Package (1)           Old Version        New Version        Net Change

numix-icon-theme-git  0.r1763.292f2b2-1  0.r1763.292f2b2-1    0,00 MiB

Total Installed Size:  24,47 MiB
Net Upgrade Size:       0,00 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                                    [##############################################] 100%
(1/1) checking package integrity                                                  [##############################################] 100%
(1/1) loading package files                                                       [##############################################] 100%
(1/1) checking for file conflicts                                                 [##############################################] 100%
(1/1) checking available disk space                                               [##############################################] 100%
:: Processing package changes...
(1/1) reinstalling numix-icon-theme-git                                           [##############################################] 100%
:: Installing vundle-git cached package...
loading packages...
warning: numix-icon-theme-git-0.r1763.292f2b2-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Package (1)           Old Version        New Version        Net Change

numix-icon-theme-git  0.r1763.292f2b2-1  0.r1763.292f2b2-1    0,00 MiB

Total Installed Size:  24,47 MiB
Net Upgrade Size:       0,00 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                                    [##############################################] 100%
(1/1) checking package integrity                                                  [##############################################] 100%
(1/1) loading package files                                                       [##############################################] 100%
(1/1) checking for file conflicts                                                 [##############################################] 100%
(1/1) checking available disk space                                               [##############################################] 100%
:: Processing package changes...
(1/1) reinstalling numix-icon-theme-git                                           [##############################################] 100%
:: Installing powerline-fonts-git cached package...
loading packages...
warning: numix-icon-theme-git-0.r1763.292f2b2-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Package (1)           Old Version        New Version        Net Change

numix-icon-theme-git  0.r1763.292f2b2-1  0.r1763.292f2b2-1    0,00 MiB

Total Installed Size:  24,47 MiB
Net Upgrade Size:       0,00 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                                    [##############################################] 100%
(1/1) checking package integrity                                                  [##############################################] 100%
(1/1) loading package files                                                       [##############################################] 100%
(1/1) checking for file conflicts                                                 [##############################################] 100%
(1/1) checking available disk space                                               [##############################################] 100%
:: Processing package changes...
(1/1) reinstalling numix-icon-theme-git                                           [##############################################] 100%
:: Installing oh-my-zsh-git cached package...
loading packages...
warning: numix-icon-theme-git-0.r1763.292f2b2-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Package (1)           Old Version        New Version        Net Change

numix-icon-theme-git  0.r1763.292f2b2-1  0.r1763.292f2b2-1    0,00 MiB

Total Installed Size:  24,47 MiB
Net Upgrade Size:       0,00 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                                    [##############################################] 100%
(1/1) checking package integrity                                                  [##############################################] 100%
(1/1) loading package files                                                       [##############################################] 100%
(1/1) checking for file conflicts                                                 [##############################################] 100%
(1/1) checking available disk space                                               [##############################################] 100%
:: Processing package changes...
(1/1) reinstalling numix-icon-theme-git                                           [##############################################] 100%
:: Installing numix-circle-icon-theme-git cached package...
loading packages...
warning: numix-icon-theme-git-0.r1763.292f2b2-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Package (1)           Old Version        New Version        Net Change

numix-icon-theme-git  0.r1763.292f2b2-1  0.r1763.292f2b2-1    0,00 MiB

Total Installed Size:  24,47 MiB
Net Upgrade Size:       0,00 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                                    [##############################################] 100%
(1/1) checking package integrity                                                  [##############################################] 100%
(1/1) loading package files                                                       [##############################################] 100%
(1/1) checking for file conflicts                                                 [##############################################] 100%
(1/1) checking available disk space                                               [##############################################] 100%
:: Processing package changes...
(1/1) reinstalling numix-icon-theme-git                                           [##############################################] 100%
:: Installing hardcode-fixer-git cached package...
loading packages...
warning: numix-icon-theme-git-0.r1763.292f2b2-1 is up to date -- reinstalling
resolving dependencies...
looking for conflicting packages...

Package (1)           Old Version        New Version        Net Change

numix-icon-theme-git  0.r1763.292f2b2-1  0.r1763.292f2b2-1    0,00 MiB

Total Installed Size:  24,47 MiB
Net Upgrade Size:       0,00 MiB

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                                    [##############################################] 100%
(1/1) checking package integrity                                                  [##############################################] 100%
(1/1) loading package files                                                       [##############################################] 100%
(1/1) checking for file conflicts                                                 [##############################################] 100%
(1/1) checking available disk space                                               [##############################################] 100%
:: Processing package changes...
(1/1) reinstalling numix-icon-theme-git   
```

**Comment by rmarquis** [comment]
Someone reported a somewhat [similar issue(https://bbs.archlinux.org/viewtopic.php?pid=1618610#p1618610) in the forums. This might be related.

Please provide a debug output.

**Comment by rmarquis** [comment]
Commit of interest: 89d673b61a632a22bdb07048dcab4fde62ad5969.

**Comment by ChuckDaniels87** [comment]
[Here you are](https://gist.github.com/ChuckDaniels87/655b21ae26ece6719bcf19c6150606c0).

**Comment by rmarquis** [comment]
I can't say the debug output is really helpful... and of course I cannot reproduce.

Can you tell me more about your setup? Which shell are you using, and are you using third party tool such as powerpill?

**Comment by ChuckDaniels87** [comment]
Not using any third party plugin. I've tried in bash and zsh. Additionally,
I've also tested with LANG=es_ES.UTF-8 and LANG=en_US.UTF-8.
If you want me to do any additional test just tell me.
El 10 abr. 2016 11:22, "Remy Marquis" <notifications@github.com> escribió:

I can't say the debug output is really helpful... and of course I cannot
reproduce.

Can you tell me more about your setup? Which shell are you using, and are
you using third party tool such as powerpill?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/rmarquis/pacaur/issues/480#issuecomment-207950089>

**Comment by rmarquis** [comment]
Yes please:

> Also, if you could try to revert the "commit of interest" above and see if it fix your issue, that would be great.

I need to find out which commit is responsible for the issue, and in which circumstances it does appear.

**Comment by rmarquis** [comment]
Ah, think I found the issue. It seems it happens when PKGDEST is not defined. I'll have a closer look when I have access to my machine tonight.

**Comment by rmarquis** [comment]
Fixed in 46d78d3593ab9c3b3e93e731e1b9aa0c5c591036.

**Comment by ChuckDaniels87** [comment]
That was quick ;)
